dv/uvm/core_ibex/yaml/rtl_simulation.yaml

```diff
@@ -52,6 +52,29 @@
     wave_opts: >
       -ucli -do <cwd>/vcs.tcl
 
+- tool: questa
+  compile:
+    cmd:
+      - "vmap mtiUvm $QUESTA_HOME/questasim/uvm-1.2"
+      - "vlog -64
+        -access=rwc
```

Comment:
Can you indent this line a bit more to follow the style of the xlm target for example?

Reply:
ok will do

Reply:
it's updated accordingly